_posts/guides/2022-10-11-guide_localization.md

```diff
@@ -0,0 +1,104 @@
+
+![](/assets/images/localization/loc-proofread-toggle.jpg)
+
+Similarly, we're expecting proofreaders to be testing the strings for their language (see below) to ensure the game is looking good. The Momentum Mod team will work closely with proofreaders to ensure every bit of User Interface can accommodate languages.
+
+## Testing Strings In the Game
+
+Every push to Steam that the team does will update the terms to the latest available translations from the website. However, as we have a weekly-ish release cadance, this could mean that work is done before then and would need testing to see how it may appear in-game.
+
+Inside of your `Momentum Mod Playtest/momentum/resource/` folder will be a ton of text files labeled `momentum_<language>.txt`. These are the localization files that Momentum Mod uses to apply the translations to the User Interface in the game.
+
+![](/assets/images/localization/loc-local-files.jpg)
+
+You can open the `momentum_<language>.txt` of choice and make edits inside of this file. If you want to see what a token looks like with a different value, search for the text as it appears in the game, edit the value, save the file, and reload Momentum Mod.
+
+{:.notice--warning}
+It is important to save this file in the **`UTF-8`** file format! You may need to use something other than Notepad to do so, as Notepad may default to saving it as "ASCII" which will mess up the encoding!
+
+If something's broken, then, first of all, excellent find! You can report issues with translations inside our Discord server, under that language's specific Localization forum post. The proofreaders and Momentum Mod team will investigate it further and fix it in due time.
+
+## Adding Terms (Momentum Developers Only)
+
+{:.notice--danger}
+**This section of the documentation is for Momentum Mod team developer reference only!**
+
+Adding terms is quick and easy through the POEditor page.
+
+At [the Project dashboard](https://poeditor.com/projects/view?id=156379), click the "Terms" tab and at the bottom of the page, the "Add Term" button.
+
+![](/assets/images/localization/loc-add-terms.jpg)
+
+Add the new term referenced in the code, and any context if you'd like to help explain how the term is being used.
+
+![](/assets/images/localization/loc-new-term.jpg)
+
+After the term is added, head to the English section, and give it the proper English value for it.
+
+All done!
+
+### Testing Strings from POEditor
+
+There's a helpful Python script to pull in the latest translation work, found inside `Momentum Mod Playtest/momentum/tools/`, called `get_poeditor_strings.py`.
+
+This script requires [the latest version of Python 3](https://www.python.org/downloads/), and [a readonly POEditor API token](https://poeditor.com/account/api) to be used correctly.
+
+To run this script, open a terminal and set the following variables:
+
+On Windows (PowerShell):
```

Comment:
I find it's most convenient to add this to environment variables in the Windows Control Panel so they stick around. So can you add

```
Alternatively, you can go to System Properties > Environment Variables and add to your User Variables there.
```